source/use-cases/sensitive-data-encryption.txt

```diff
@@ -124,6 +124,151 @@ In this step, we generate a local master key to expedite setup of our developmen
           }
 
 
+B. Create a Data Encryption Key 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this section, we use the MongoDB driver to create and store a data 
+encryption key, used by CSFLE to automatically encrypt and decrypt data on 
+document fields. The following configuration values are required by the client 
+to generate a new data encryption key:
+
+* The local master key or AWS KMS master key access settings.
+* A MongoDB connection string that authenticates to a running server.
+* The key vault database and collection name.
+
+Upon creating a data encryption key, the driver returns the ``id`` field of
```

Comment:
Shouldn't it be ``_id`` instead of ``id``? It also might be worth adding an admonition that the _id field of a key is *not* an ObjectID.

Reply:
I agree, I should use `_id`. Should I identify this as a `UUID` for the reader or is that too much detail?

Reply:
I think you should call it out for sure. It will be surprising that is a UUID instead of an ObjectID.

Reply:
Fixed, let me know if it looks ok now.